packages/components/src/components/DaScroll.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="scroll__switch"
+    :class="[showed ? 'shown' : '']"
+    @click="scrollTop"
+  >
+    <svgicon name="arrow" class="scroll__switch__icon" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DaScroll",
+
+  data() {
+    return {
+      showed: false,
+    };
+  },
+
+  methods: {
+    scrollTop() {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    },
+  },
+
+  created() {
+    this.scroll = (event) => {
+      this.showed =
+        document.documentElement.scrollTop >
+        document.documentElement.scrollHeight / 3;
+    };
+    this.scroll = this.scroll.bind(this);
+  },
+
+  mounted() {
+    import("../../icons/arrow");
+    window.addEventListener("scroll", this.scroll);
```

Comment:
Please make sure to use a passive event listener to improve scrolling performance.
Take a look here:
https://web.dev/uses-passive-event-listeners/

Also make sure to remove the event listener on dismount